src/System.Net.Http/src/System.Net.Http.csproj

```diff
@@ -345,16 +345,8 @@
     <Compile Include="netcore50\System\Net\HttpClientHandler.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == 'uap101'">
-    <ProjectReference Include="..\..\System.Runtime.Extensions\ref\System.Runtime.Extensions.csproj">
-      <UndefineProperties>TargetGroup;OSGroup</UndefineProperties>
-    </ProjectReference>
-    <ProjectReference Include="..\..\System.Runtime.InteropServices\ref\System.Runtime.InteropServices.csproj">
-      <UndefineProperties>TargetGroup;OSGroup</UndefineProperties>
-    </ProjectReference>
-    <ProjectReference Include="..\..\System.Text.Encoding\ref\System.Text.Encoding.csproj">
-      <UndefineProperties>TargetGroup;OSGroup</UndefineProperties>
-    </ProjectReference>
-    <ProjectReference Include="..\..\System.Globalization.Extensions\ref\System.Globalization.Extensions.csproj">
+    <!-- remove once we have new packages https://github.com/dotnet/corefx/issues/12838 -->
```

Comment:
Sure.  I'll need to rebase this once https://github.com/dotnet/corefx/pull/13542 goes through so I'll take care of it then.